contracts/bridge/BridgeOperator.sol

```diff
@@ -34,6 +34,7 @@ contract BridgeOperator is Ownable {
     mapping(uint8 => mapping (uint64 => VotesData)) private votes; // <voteType, <nonce, VotesData>
     mapping(uint64 => bool) public closedValueTransferVotes; // <nonce, bool>
 
+    uint64 public constant MAX_OPERATOR = 12;
```

Comment:
Is there a special reason to limit the maximum number of operators to 12?

Reply:
There is no special reason. 
But if we add one more operator, it need 5455 gas for removing the vote data. :)